exercises/tower-of-hanoi/hanoi.go

```diff
@@ -0,0 +1,30 @@
+package towerofhanoi
+
+// This is an stub file. Remove this comment and
```

Comment:
```suggestion
// This is a stub file...
```

Reply:
Added as https://github.com/exercism/go/pull/1272/commits/bc507303b28de3db175928d057303032d4b8c3f2

Unsure whether you suggested the full change or just the grammar issue.